docs/05-technical-reference/ac-01-application-connectivity-api-specification.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Application Connectivity API specification
```

Comment:
Content to be added in the next iteration, is that correct?

Reply:
Yes